spec/system/candidate_interface/submitting/candidate_tries_to_submit_to_undergraduate_course_without_a_levels_spec.rb

```diff
@@ -0,0 +1,98 @@
+require 'rails_helper'
+
+RSpec.describe 'Candidate tries to submit undergraduate courses with no A levels' do
+  include CandidateHelper
+
+  scenario 'when teacher degree apprenticeship is live' do
+    given_i_am_on_the_cycle_when_candidates_can_enter_details_for_undergraduate_course
+    and_teacher_degree_apprenticeship_feature_flag_is_on
+    and_i_am_signed_in
+    when_i_view_the_a_levels_section
+    then_i_see_the_content_for_postgraduate_and_undergraduate
+    when_i_choose_no_a_levels
+    and_i_click_continue
+    then_i_am_on_a_levels_review_page
+    and_i_see_the_postgraduate_and_undergraduate_content_for_no_a_levels
+  end
+
+  scenario 'when teacher degree apprenticeship is not live' do
+    given_i_am_on_the_cycle_when_candidates_can_not_enter_details_for_undergraduate_course
+    and_i_am_signed_in
+    when_i_view_the_a_levels_section
+    then_i_see_the_content_for_postgraduate
+
+    when_i_choose_no_a_levels
+    and_i_click_continue
+    then_i_am_on_a_levels_review_page
+    and_i_see_the_postgraduate_content_for_no_a_levels
```

Comment:
TL;DR The submitting steps will be done in the next card/PR.


If you are wondering that the spec is under submitting and only has A levels section, I consciously optin to test the A level requirement as part of this spec and prepare the ground for the [next card](https://trello.com/c/ph1ZE5tm/2089-tda-apply-prevent-candidate-from-reviewing-their-application-without-at-least-one-qualification-in-a-levels-and-other-qualificat) which I will add more steps here.

The A levels section are totally related to the content that it shows during the submission so it makes sense to have all under one spec.